keps/sig-apps/4212-declarative-node-maintenance/kep.yaml

```diff
@@ -0,0 +1,52 @@
+title: Declarative Node Maintenance
+kep-number: 4212
+authors:
+  - "@atiratree"
+owning-sig: sig-apps
```

Comment:
Actually: why Apps as the **owning** SIG? I'd have thought Node or maybe Cluster Lifecycle.

Reply:
There are a lot of areas that this KEP touches on and will need cooperation across. But I think the middle ground is the safety of applications and how it affects workloads and their controllers - I want to add a section on DaemonSets. Evictions and PDBs are close to `EvacuationRequest` and need to be considered as well. So in my opinion it is best suited for sig-apps so far.